README.md

```diff
@@ -515,17 +518,6 @@ If the user is not logged the return will also be `false`.
 
 ## Troubleshooting
 
-If you encounter an error when doing the migration that looks like:
-
-```
-SQLSTATE[HY000]: General error: 1005 Can't create table 'laravelbootstrapstarter.#sql-42c_f8' (errno: 150)
-    (SQL: alter table `role_user` add constraint role_user_user_id_foreign foreign key (`user_id`)
-    references `users` (`id`)) (Bindings: array ())
-```
-
-Then it's likely that the `id` column in your user table does not match the `user_id` column in `role_user`.
-Make sure both are `INT(10)`.
-
```

Comment:
Some of these changes to the readme file aren't necessary like removing of this and model namespacing.